packages/datepicker/src/ux-calendar.css

```diff
@@ -0,0 +1,55 @@
+ux-calendar {
+  display: block;
+  height: 260px;
+  display: flex;
+  flex-direction: column;
+}
+
+ux-calendar div.calendar-row {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 2em;
+}
+
+ux-calendar .calendar-row.weekdays {
+  color: var(--ux-theme--datepicker-weekday-foreground, var(--swatch-grey-p600));
```

Comment:
What does the nested var expression do?

Reply:
With css variables, to make a fallback that is also a variable, it has to be wrapped in a var().

var(/*variable name*/, /*fallback property*/) 

So if a fallback property is also a css variable, it must also be wrapped.